resin_release_tool/cli.py

```diff
@@ -118,5 +118,50 @@ def releases(releaser, count):
         click.echo(f'{release["end_timestamp"]} {release["commit"]}')
 
 
+@cli.command()
+@pass_releaser
+def show_devices(releaser, devices_to_show):
+    devices_to_show= ', '.join(
+        [c['uuid'][:6] for c in devices_to_show.values()])
+    click.echo(f'uuids: {devices_to_show}')
+
+
+@cli.command()
+@click.argument('envar_model')
+@click.argument('envar_name')
+@click.argument('envar_value')
+@click.option('--exclusive', is_flag=True, default=False)
+@pass_releaser
+@click.pass_context
+def filter_and_remove_env_var(ctx, releaser, envar_model, \
+                              envar_name, envar_value, exclusive):
+    """Filter devices by enviroment variables and removes them."""
```

Comment:
Can you add more documentation here, like that each variable means and help to exclusive?

Reply:
Sure! On it